Source/WTF/wtf/text/StringCommon.h

```diff
@@ -913,7 +913,7 @@ inline void copyElements(ElementType* __restrict destination, const ElementType*
         std::memcpy(destination, source, length * sizeof(ElementType));
 }
 
-inline void copyElements(uint16_t* __restrict destination, const uint8_t* __restrict source, size_t length)
+inline void copyElements16To8(uint16_t* __restrict destination, const uint8_t* __restrict source, size_t length)
```

Comment:
When we have correct code, but we need to suppress it in order to work around a limitation in a particular compiler, the best approach is to #ifdef or #ifndef for that compiler.

You can see this in cases like #ifdef __cplusplus, #ifdef __OBJC__, #if !COMPILER(MSVC), and so on.

#ifdef for the errant or un-supporting compiler is more sustainable in a project that supports many compilers, and it's more self-documenting.